docs/apache-airflow-providers-amazon/auth-manager/setup/config.rst

```diff
@@ -0,0 +1,36 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+====================================
+Set AWS auth manager as auth manager
```

Comment:
I don't exactly have a better suggestion (I'll keep thinking on it) but this sounds a bit confusing.

Reply:
Maybe "Use AWS auth manager in Airflow". It am not entirely convinced either :/

Reply:
> Maybe "Use AWS auth manager in Airflow". It am not entirely convinced either :/

Yeah I like that  :+1:

Reply:
I actually decided to go with "Configure Airflow to use the AWS auth manager"